platform/backend/repository/record/src/main/kotlin/io/hamal/repository/record/func/Entity.kt

```diff
@@ -39,7 +43,24 @@ data class FuncEntity(
                 sequence = rec.sequence(),
                 name = rec.name,
                 inputs = rec.inputs,
-                code = rec.code
+                code = FuncCode(
+                    id = rec.codeId,
+                    version = rec.codeVersion,
+                    deployedVersion = code!!.deployedVersion
+                )
+            )
+
+            is FuncDeploymentRecord -> copy(
+                id = rec.entityId,
+                cmdId = rec.cmdId,
+                sequence = rec.sequence(),
+                name = name,
+                inputs = inputs,
+                code = FuncCode(
+                    id = code!!.id,
+                    version = code!!.version,
+                    deployedVersion = rec.deployment
```

Comment:
```suggestion
                    deployedVersion = rec.deployedVersion
```

We need a better name for the variable